eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Deleg.hs

```diff
@@ -193,16 +203,6 @@ conwayDelegTransition = do
             DelegStake targetPool -> checkPoolRegistered targetPool
             DelegStakeVote targetPool _ -> checkPoolRegistered targetPool
             DelegVote _ -> pure ()
-    -- Whenever we want to accept new deposit, we must always check if the stake credential isn't already registered.
```

Comment:
This comment got omitted in the refactoring, but i think the name of the function `checkStakeKeyNotRegistered` is explanatory enough.

Reply:
This comment got removed, because it no longer happens. This check got inlined by extracting it out of the function that called `checkStakeKeyNotRegistered`